tests/unit/amo/server/test_base.js

```diff
@@ -165,10 +165,13 @@ describe(__filename, () => {
     });
 
     it('preconnects to addons-server CDN', async () => {
-      const config = getFakeConfig({
-        staticHost: undefined,
-        amoCDN: 'https://cdn.example.com',
-      });
+      const config = getFakeConfig(
+        {
+          staticHost: undefined,
+          amoCDN: 'https://cdn.example.com',
+        },
+        { allowUnknownKeys: true },
```

Comment:
Why is this needed?

Reply:
Because `staticHost` is `undefined` by default, so the `config.has(key)` in `getFakeConfig` fails when you try to set it :(

Reply:
I removed it in a few tests in https://github.com/mozilla/addons-frontend/pull/10893/commits/e0b2c0967b9f08ed386394469ba9c118d3a32984 but it fails without it in the rest.